__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const { app } = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api", () => {
+  describe("/api/categories", () => {
+    test("200: returns an array of category objects", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((response) => {
+          const categories = response.body.categories;
+          expect(categories.length === 4).toBe(true);
+          categories.forEach((category) => {
+            expect(category.hasOwnProperty("slug")).toBe(true);
+            expect(category.hasOwnProperty("description")).toBe(true);
+          });
+        });
+    });
+    test("404: returns an error message when passed an invalid url path", () => {
```

Comment:
Keep this 404 test for mispellings, but you won't need to make any more for other end points.  You could put it in its own describe block since it's not unique to this end-point